openslides_backend/action/motion/set_recommendation.py

```diff
@@ -0,0 +1,43 @@
+from typing import Any, Dict
+
+from ...models.models import Motion
+from ...shared.exceptions import ActionException
+from ...shared.patterns import Collection, FullQualifiedId
+from ..default_schema import DefaultSchema
+from ..generics import UpdateAction
+from ..register import register_action
+
+
+@register_action("motion.set_recommendation")
+class MotionSetRecommendationAction(UpdateAction):
+    """
+    Set a recommendation in a motion.
+    """
+
+    model = Motion()
+    schema = DefaultSchema(Motion()).get_update_schema(properties=["recommendation_id"])
+
+    def update_instance(self, instance: Dict[str, Any]) -> Dict[str, Any]:
+        """
+        Check recommendation workflow_id and recommendation_label.
+        """
+        if instance.get("recommendation_id") is not None:
```

Comment:
Since it's required, you don't need this check.

Reply:
Removed.